src/Samples/Net5ProjectionSample/SimpleMathProjection/nuget/SimpleMathProjection.nuspec

```diff
@@ -9,17 +9,17 @@
 			<group targetFramework=".NETCoreApp3.0" />
 			<group targetFramework="UAP10.0" />
 			<group targetFramework=".NETFramework4.6" />
-			<group targetFramework="net5.0">
-				<dependency id="Microsoft.Windows.CsWinRT" version="1.0.1" exclude="Build,Analyzers" />
-			</group>
+			<group targetFramework="net5.0-windows10.0.17763.0" />
 		</dependencies>
 	</metadata>
 	<files>
 		<!--Support net46+, netcore3, net5, uap, c++ -->
+    <!--Architecture-netural assemblies-->
 		<file src="..\..\_build\x64\Debug\SimpleMathComponent\bin\SimpleMathComponent\SimpleMathComponent.winmd" target="lib\netcoreapp3.0\SimpleMathComponent.winmd" />
 		<file src="..\..\_build\x64\Debug\SimpleMathComponent\bin\SimpleMathComponent\SimpleMathComponent.winmd" target="lib\uap10.0\SimpleMathComponent.winmd" />
 		<file src="..\..\_build\x64\Debug\SimpleMathComponent\bin\SimpleMathComponent\SimpleMathComponent.winmd" target="lib\net46\SimpleMathComponent.winmd" />
-		<file src="..\..\_build\x64\Debug\SimpleMathProjection\bin\SimpleMathProjection.dll" target="lib\net5.0\SimpleMathProjection.dll" />
-		<file src="..\..\_build\x64\Debug\SimpleMathComponent\bin\SimpleMathComponent\SimpleMathComponent.dll" target="runtimes\win10-x64\native\SimpleMathComponent.dll" />
+		<file src="..\..\_build\x64\Debug\SimpleMathProjection\bin\SimpleMathProjection.dll" target="lib\net5.0-windows10.0.17763.0\SimpleMathProjection.dll" />
```

Comment:
Is 17763 accurate here given the TFM is 19041?

Reply:
Changed the nuspec TFM to 19041, and TPMV to 17763